GOVERNANCE.md

```diff
@@ -0,0 +1,192 @@
+
+Responsibilities include:
+
+- Strong commitment to the project
+- Participate in design and technical discussions
+- Participate in the conflict resolution and voting process at the repository scope when necessary
+- Seek review and obtain approval from the steering committee when making a change to central architecture that will have broad impact across multiple repositories
+- Contribute non-trivial pull requests
+- Perform code reviews on other's pull requests
+- Ensure that proposed changes to your repository adhere to the established standards, best practices, and guidelines, and that the overall quality and integrity of the code base is upheld.
+- Add and remove maintainers to the repository as described below
+- Approve and merge pull requests into the code base
+- Regularly triage GitHub issues. The areas of specialization possibly listed in OWNERS.md can be used to help with routing an issue/question to the right person.
+- Make sure that ongoing PRs are moving forward at the right pace or closing them
+- Monitor Atlantis Slack (delayed response is perfectly acceptable), particularly for the area of your repository
+- Regularly attend the recurring community meetings
+- Periodically attend the recurring steering committee meetings to provide input
+- In general, continue to be willing to spend at least 25% of their time working on Atlantis (~1.25 business days per week)
+
+Each repository's current list of maintainers is published and updated in each repo’s OWNERS.md file.
+
+#### Removing a core-contributor or maintainer
+
+If a core-contributor or maintainer is no longer interested or cannot perform the the duties listed above, they should volunteer to be moved to emeritus status. In extreme cases this can also occur by a vote of the maintainers per the voting process below.
+
+## Conflict resolution and voting
+In general, it is preferred that technical issues and team membership are amicably worked out between the persons involved. If a dispute cannot be decided independently, the leadership at the appropriate scope can be called in to decide an issue. If that group cannot decide an issue themselves, the issue will be resolved by voting.
+
+### Issue Voting Scopes
+Issues can be resolved or voted on at different scopes:
+
+* **Repository**: When an issue or conflict only affects a single repository, then the maintainer team for that repository should resolve or vote on the issue. This includes technical decisions as well as team membership.
+* **Organization**: If an issue or conflict affects multiple repositories or the Atlantis organizations and community at large, the steering committee should resolve or vote on the issue.
+
+### Issue Voting Process
+
+The issue voting process is usually a simple majority in which each entity within the voting scope gets a single vote. The following decisions require a super majority (at least 2/3 of votes). All other decisions and changes require only a simple majority:
```

Comment:
what happens if voting is 50/50 (even number of legible voters) - who acts as tie breaker?

Reply:
There's no guidance on this. See: https://github.com/cncf/project-template/blob/main/GOVERNANCE-maintainer.md#voting

The voting entities should prioritize membership count with odd numbers to avoid this, and most likely will be to prioritize nominating/voting new members to the role before proceeding with a vote.